.github/workflows/examples.yml

```diff
@@ -0,0 +1,38 @@
+name: examples
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - '.github/workflows/examples.yml'
+      - './examples/go.mod'
+      - './examples/go.sum'
+      - './tutorials/go.mod'
+      - './tutorials/go.sum'
+      - './go.mod'
+      - './go.sum'
```

Comment:
Is this intended?

Reply:
yep, if we change something in the root module, since the other two replace it, we should tidy them as well